sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 * Added method `get_topic_sender` in `ServiceBusClient` to get a `ServiceBusSender` for a topic.
 * Added method `get_subscription_receiver` in `ServiceBusClient` to get a `ServiceBusReceiver` for a subscription under specific topic.
 * `ServiceBusSender.send()` can now send a list of messages in one call, if they fit into a single batch.  If they do not fit a `ValueError` is thrown.
+* `BatchMessage.add()` and `ServiceBusSender.send()` raises `MessageContentTooLarge`, which is a subclass of `ValueError` if the content is over-sized.
+* `ServiceBusReceiver.receive()` raises `ValueError` if the max_batch_size is greater than the prefetch of `ServiceBusClient`.
+* Added exception classes `MessageError`, `MessageContentTooLarge`, `ServiceBusAuthenticationError`.
```

Comment:
Probably add some description here?

* Added exception classes `MessageError`, `MessageContentTooLarge`, `ServiceBusAuthenticationError`.
  - `MessageError`: general message error type?
  - `MessageContentTooLarge`: message over-side, subclass of `ValueError`
  - `ServiceBusAuthenticationError`: fail to be authenticated by the service?

Reply:
Change log update will be in a separate PR.